fe1-web/src/features/home/screens/Launch.tsx

```diff
@@ -39,19 +40,19 @@ const Launch: FunctionComponent<IPropTypes> = ({ navigation }) => {
     }
 
     getNetworkManager().connect(inputAddress);
+
     createLao(laoName)
-      .then((channel: Channel) =>
+      .then((channel: Channel) => {
         subscribeToChannel(channel).then(() => {
```

Comment:
here adding a `{` changed the logic significantly.
It switched from chaining promises (i.e. `createLao` + `subscribe`) to resolving the promise after `createLao` and continuing with `subscribe` asynchronously.
Is it a problem?

Reply:
I am honestly not sure where that `{` comes from. Thank you for pointing it out! 👍

It is not a problem here (last promise in the chain and the return value is ignored) but can quickly become one if one does not see the `{`